tf-modules/ingest/lambda-functions.tf

```diff
@@ -299,6 +298,36 @@ resource "aws_lambda_function" "sqs2sfThrottle" {
   tags = var.tags
 }
 
+resource "aws_lambda_function" "sqs2sfThrottleRateLimited" {
+  function_name    = "${var.prefix}-sqs2sfThrottleRateLimited"
+  filename         = "${path.module}/../../packages/api/dist/sfStarter/lambda.zip"
+  source_code_hash = filebase64sha256("${path.module}/../../packages/api/dist/sfStarter/lambda.zip")
+  handler          = "index.sqs2sfThrottleRateLimitedHandler"
+  role             = var.lambda_processing_role_arn
+  runtime          = "nodejs20.x"
+  # This timeout must match the cadence of the event bridge rule that triggers it
+  timeout          = 60
```

Comment:
Had a brief conversation offline with @mikedorfman  re: this, I think there's a potential timing issue with this timeout being too close to the event loop in clearing sempahores meaning the lambda could end without clearing semaphore.   We talked through it and it's *probably* the right call to extend the timeout and make sure the event loop is configured such that it stops dispatching based on the EB periodicity but the lambda has a much longer timeout to allow things like semaphores to clear. 

   Noting here to track changes/etc.